tracim/setup.py

```diff
@@ -1,65 +1,87 @@
 # -*- coding: utf-8 -*-
-#quickstarted Options:
-#
-# sqlalchemy: True
-# auth:       sqlalchemy
-# mako:       True
-#
-#
+# quickstarted Options:
+#  sqlalchemy: True
+#  auth:       sqlalchemy
+#  mako:       True
 
-#This is just a work-around for a Python2.7 issue causing
-#interpreter crash at exit when trying to log an info message.
+import sys
+
+from setuptools.command.develop import develop
+# This is just a work-around for a Python2.7 issue causing
+# interpreter crash at exit when trying to log an info message.
 try:
     import logging
     import multiprocessing
 except:
     pass
 
-import sys
-py_version = sys.version_info[:2]
-
 try:
     from setuptools import setup, find_packages
 except ImportError:
     from ez_setup import use_setuptools
     use_setuptools()
     from setuptools import setup, find_packages
 
+py_version = sys.version_info[:2]
+
+DESCRIPTION = ('Tracim is a plateform software designed to improve '
+               'traceability and productivity in collaborative work.')
+
```

Comment:
You moved the description on a global var which is used only once. This is overkill.

By the way, it is a good habit to write 
```
DESCRIPTION = ('Tracim is a plateform software designed to improve '
               'traceability and productivity in collaborative work.')
```

as the following:

```
DESCRIPTION = (
    'Tracim is a plateform software designed to improve '
    'traceability and productivity in collaborative work.'
)
```

The reason is that the first way to write will be broken on name-refactoring (which is a relatively common operation)